kdt-spring-order/src/main/java/org/prgrms/kdt/voucher/VoucherFactory.java

```diff
@@ -0,0 +1,61 @@
+package org.prgrms.kdt.voucher;
+
+import org.prgrms.kdt.entity.FixedAmountVoucher;
+import org.prgrms.kdt.entity.PercentDiscountVoucher;
+import org.prgrms.kdt.entity.Voucher;
+import org.springframework.stereotype.Component;
+
+import java.util.NoSuchElementException;
+import java.util.UUID;
+
+@Component
+public class VoucherFactory {
+
+    private Voucher createFixedAmountVoucher(String discountValue) {
+        return new FixedAmountVoucher(UUID.randomUUID(), Long.parseLong(discountValue));
+    }
+
+    private Voucher createPercentDiscountVoucher(String discountValue) {
+        return new PercentDiscountVoucher(UUID.randomUUID(), Double.parseDouble(discountValue));
+    }
+
+    private Voucher createFixedAmountVoucher(String discountValue, String voucherId) {
+        return new FixedAmountVoucher(UUID.fromString(voucherId), Long.parseLong(discountValue));
+    }
+
+    private Voucher createPercentDiscountVoucher(String discountValue, String voucherId) {
+        return new PercentDiscountVoucher(UUID.fromString(voucherId), Double.parseDouble(discountValue));
+    }
+
+    public Voucher createVoucher(String voucherType, String discountValue) {
+        String classNameString = VoucherType.getStringClassName(voucherType);
+
+        if (classNameString.equals("FixedAmountVoucher")) {
+            return createFixedAmountVoucher(discountValue);
+        } else if (classNameString.equals("PercentDiscountVoucher")) {
+            return createPercentDiscountVoucher(discountValue);
+        } else {
+            throw new NoSuchElementException("존재하지 않는 Voucher 형식입니다.");
+        }
```

Comment:
```suggestion
            return createPercentDiscountVoucher(discountValue);
        }

        throw new NoSuchElementException("존재하지 않는 Voucher 형식입니다.");
```
else 문 안써도 될 듯합니다. 테스트 해보시죠!